core/javatests/com/google/census/CensusScopeTest.java

```diff
@@ -41,58 +41,76 @@ public void testNoScope() {
 
   @Test
   public void testScope() {
-    try (CensusScope scope = new CensusScope(Census.getCurrent().with(K1, V1))) {
+    CensusScope scope = new CensusScope(Census.getCurrent().with(K1, V1));
```

Comment:
If Census supports Java 6, then I think we should also run the tests with it.  I'd like to add the tests to pom.xml soon.